vision_api_interface.py

```diff
@@ -251,6 +252,7 @@ def get_session_id(create_session_response):
         :return session_id: Session ID
         """
         create_session_text = json.loads(create_session_response.text)
+        print("API Response (create_session_response):", create_session_response.text)
```

Comment:
Any reason not to use `create_session_text` here?
```suggestion
        print("API Response (create_session_response):", create_session_text)
```